package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "bitmovin-player-react-native-analytics-conviva",
-  "version": "1.3.0",
+  "version": "1.4.0",
```

Comment:
please revert this
```suggestion
  "version": "1.3.0",
```

Reply:
[912f10f](https://github.com/bitmovin/bitmovin-player-react-native-analytics-conviva/pull/29/commits/912f10f218efd80288b98bb9b0a6f210b670b19d)